lib/Db/Feed.php

```diff
@@ -660,4 +662,27 @@ public function toAPI(): array
             ]
         );
     }
+
+    public function toAPI2(bool $reduced = false): array
+    {
+        $result = [
+            'id' => $this->getId(),
+            'name' => $this->getTitle(),
+            'faviconLink' => $this->getFaviconLink(),
+            'folderId' => $this->getFolderId(),
+            'ordering' => $this->getOrdering(),
+            'fullTextEnabled' => $this->getFullTextEnabled(),
+            'updateMode' => $this->getUpdateMode(),
+            'isPinned' => $this->getPinned()
+        ];
+
+        if (!empty($this->getLastUpdateError())) {
```

Comment:
I don't think the latter should happen but I think we should catch it regardless
```suggestion
        if (is_null($this->getLastUpdateError()) || trim($this->getLastUpdateError()) === '') {
```